src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx

```diff
@@ -91,9 +91,13 @@ const SelectableAssetItem = ({tokenInfo, disabled, wallet}: SelectableAssetItemP
     tokenSelectedChanged(tokenInfo.id)
 
     // if the balance is atomic there is no need to edit the amount
-    if (Quantities.isAtomic(spendable, tokenInfo.decimals)) {
+    if (tokenInfo.kind === 'ft' && Quantities.isAtomic(spendable, tokenInfo.metadata.decimals)) {
       amountChanged(spendable)
       navigation.navigate('send-list-amounts-to-send')
+    }
+    if (tokenInfo.kind === 'nft') {
+      amountChanged('1')
```

Comment:
Sharing what @stackchain commented here https://github.com/Emurgo/yoroi-mobile/pull/2440#discussion_r1171975709

This is my solution: https://github.com/Emurgo/yoroi-mobile/blob/2ecdba49134b40271bfb5a54f1afd94713cc9f0e/src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx#L97